partner_title_order/tests/test_partner_title.py

```diff
@@ -14,7 +14,7 @@ def test_default_order(self):
         partner_titles = self.env["res.partner.title"].search([])
         self.assertEqual(
             partner_titles.mapped("name"),
-            partner_titles.sorted(lambda x: x.name).mapped("name"),
```

Comment:
Was this change because the test failed ?
Because it should not have all sequence fields should have been set to 10 ?

Reply:
Otherwise looks good